tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/OptimizerCore.scala

```diff
@@ -2168,6 +2168,18 @@ private[optimizer] abstract class OptimizerCore(
     }
   }
 
+  private def literalToString(l: Literal): String = l match {
+    case StringLiteral(str) => str
```

Comment:
In general, we prefer to use the name of the field [as declared in the case class](https://github.com/scala-js/scala-js/blob/v0.6.5/ir/src/main/scala/org/scalajs/core/ir/Trees.scala#L738) unless there is a good reason not to (so `value` here instead of `str`).